tests/opensource-suites/dash/base/relations.test.ts

```diff
@@ -85,7 +73,7 @@ datalensTest.describe('Dashboards - Basic functionality', () => {
                 return elems.length === 1;
             });
 
-            await deleteEntity(page, WorkbooksUrls.E2EWorkbook);
+            await dashboardPage.deleteDash();
```

Comment:
Maybe you can use `afterEach` callback for deleting entity. Its looks more safety

Reply:
The flag isEnabledHideOldRelations allows you to complete the test without creating a dash, which prevents you from using this hook